Source/Meadow.Foundation.Core/Leds/Led.Extensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
```

Comment:
Let's rename this file to something like Led.Animations.cs so it's not an extension anymore